spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/rdd/CassandraTableScanRDD.scala

```diff
@@ -295,15 +420,17 @@ object CassandraTableScanRDD {
     implicit
       keyCT: ClassTag[K],
       valueCT: ClassTag[V],
-      rrf: RowReaderFactory[(K, V)]): CassandraTableScanRDD[(K, V)] = {
+      rrf: RowReaderFactory[(K, V)],
+      rwf: RowWriterFactory[K]): CassandraTableScanRDD[(K, V)] = {
 
-    new CassandraTableScanRDD[(K, V)](
+    val rdd = new CassandraTableScanRDD[(K, V)](
```

Comment:
It doesn't :( I also don't think we use this interface anywhere. I just wanted to try this and I left it in. I"ll remove it